conf/messages

```diff
@@ -270,16 +277,26 @@ label.you_may_have_previously_activated_your_self_assessment_using_different_=Yo
 label.check_your_self_assessment_details=Check your Self Assessment details.
 label.if_you_send_your_tax_return_online=If you send your tax return online
 label.if_you_send_your_tax_return_by_post=If you send your tax return by post
-label.you_cannot_currently_see_your_self_assessment_=You cannot currently see your Self Assessment details online.<br /><br /><a href="{0}" target="_blank" rel="noopener noreferrer">Send your tax return by post (opens in a new window)</a>.
-label.you_cannot_access_this_service_because_hmrc_doesnt_know_enough_=You cannot access this service because HM Revenue and Customs (HMRC) does not know enough about you to be able to confirm your identity.<br /><br />If you have a query you can <a href="{0}" target="_blank" rel="noopener noreferrer">contact HMRC (opens in a new window)</a> to get help.
+label.you_cannot_currently_see_your_self_assessment_=You cannot currently see your Self Assessment details online.<br /><br /><a href="{0}" target="_blank" rel="noopener noreferrer">Send your tax return by post (opens in a new window or tab)</a>.
+label.you_cannot_access_this_service_because_hmrc_doesnt_know_enough_=You cannot access this service because HM Revenue and Customs (HMRC) does not know enough about you to be able to confirm your identity.<br /><br />If you have a query you can <a href="{0}" target="_blank" rel="noopener noreferrer">contact HMRC (opens in a new window or tab)</a> to get help.
```

Comment:
Is it intentional to leave the markup in these entries? - Seems like you cleaned it up elsewhere :)